pennylane/beta/queuing/measure.py

```diff
@@ -58,11 +59,8 @@ def __init__(self, return_type, obs=None, wires=None, eigvals=None):
 
             self._eigvals = np.array(eigvals)
 
-        if wires is not None:
-            if obs is not None:
-                raise ValueError("Cannot set the wires if an observable is provided.")
-
-            self._wires = wires
+        if wires is not None and obs is not None:
```

Comment:
Checking whether `wires` is `None` is no longer needed (from above code, it can never be)

Reply:
Good point! I've moved this check up a few lines so that it is still relevant.